content/docs/user-guide/integrations/databricks.md

```diff
@@ -0,0 +1,47 @@
+# Databricks
+
+As of September 2023 Databricks doesn't expose the underlying GIT repo in your
+project, so GIT-related DVC functionality within the repo provided by Databricks
+is not supported (e.g. [experiments], `--rev/--all-commits/--all-tags/etc`). But
+everything will operate as normal if you `git clone` a project yourself or use
+remote projects with DVC directly.
+
+## Install
+
+```bash
+%pip install dvc
+```
+
+## DVC API
+
+You can use your existing DVC projects through [Python API] as normal.
```

Comment:
Can we add an example here like the one in `Secrets` but with publicly available data?